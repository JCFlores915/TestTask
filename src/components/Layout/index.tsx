import './index.scss'
import React from 'react'
import { Layout } from 'antd' 

const { Content: AntContent } = Layout

const Content = ({ children } : {children: React.ReactNode}) => {
  return (<div className="layout">{children}</div>)
}

export default Content