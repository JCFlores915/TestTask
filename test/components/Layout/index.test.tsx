import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../../../src/components/Layout';

describe('Layout Component', () => {
    it('renders children correctly', () => {
        const children = <div>Test Content</div>;
        const tree = renderer.create(<Layout>{children}</Layout>).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
});
