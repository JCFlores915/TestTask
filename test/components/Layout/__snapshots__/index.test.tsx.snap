// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component renders children correctly 1`] = `
<main
  className="ant-layout-content layout css-dev-only-do-not-override-zl9ks2"
>
  <div>
    Test Content
  </div>
</main>
`;
